src/components/InputText/InputText.css

```
@@ -0,0 +1,14 @@
+.Input-Text{
```

Pascal Case needed here too
```suggestion
.InputText{
```

---

@Mubangizi this has been fixed.